tests/conftest.py

```diff
@@ -1821,3 +1821,42 @@ def fixture_store_with_users(store: Store, helpers: StoreHelpers) -> Store:
     store.commit()
 
     yield store
+
+
+@pytest.fixture(name="store_with_cases_and_customers")
+def fixture_store_with_cases_and_customers(store: Store, helpers: StoreHelpers) -> Store:
+    """Return a store with cases and customers."""
+
+    customer_details = [
+        ("cust000", "Customer 1", True),
+        ("cust001", "Customer 2", False),
+        ("cust002", "Customer 3", True),
+    ]
+    customers = []
+
+    for customer_id, customer_name, scout_access in customer_details:
+        customer: Customer = helpers.ensure_customer(
+            store=store, customer_id=customer_id, name=customer_name, scout_access=scout_access
+        )
+        customers.append(customer)
+
+    case_details = [
```

Comment:
Add type hint

Reply:
I'm not sure what value these type hints add, but fixed it!